src/requests/addresses/get.ts

```diff
@@ -0,0 +1,153 @@
+import { urlQueryBuilder } from "@netsu/js-utils";
+import { UserIdentifierModel } from "../../models";
+import { axiosConf, wdData } from "../../setup";
+import { URL } from "./config";
+import { 
+	GetAddressInformationModel, 
+	GetForwardedAddressesInformationModel, 
+	ListAddressFromCommunicationRegisterResultsModel, 
+	ListRegisteredAddressesForUserModel, 
+	ListRegisteredAddressesModel, 
+	RequestAddressesInformationModel 
+} from './models';
+import { DefaultResponseModel } from '../../models/index';
+import {} from "./models";
+
+/**
+ * Request forwarded Addresses information
+ *
+ * https://docs.wildduck.email/api/#operation/getForwardedAddress
+ *
+ * @param addressId ID of the Address
+ */
+export const getForwardedAddressesInformation = async (
+	addressId: string
+): Promise<GetForwardedAddressesInformationModel> => {
+	const url = urlQueryBuilder(`${URL}/forwarded/${addressId}`, {
+		access_token: wdData.accessToken,
+	});
+
+	const res = await axiosConf.get(url);
+
+	return res.data;
+};
+
+/**
+ * Request Addresses information
+ *
+ * https://docs.wildduck.email/api/#operation/getForwardedAddress
+ *
+ * @param userId ID of the User
+ * @param addressId ID of the Address
+ */
+export const requestAddressesInformation = async (
+	userId: string,
+	addressId: string
+): Promise<RequestAddressesInformationModel> => {
+	const url = urlQueryBuilder(`/user/${userId}/${addressId}`, {
+		access_token: wdData.accessToken,
+	});
+
+	const res = await axiosConf.get(url);
+
+	return res.data;
+};
+
+/**
+ * Get Address info
+ *
+ * https://docs.wildduck.email/api/#operation/resolveAddress
+ *
+ * @param addressId ID of the Address or e-mail address string
+ * @param allowWildcard If true then resolves also wildcard addresses
+ */
+export const getAddressInformation = async (
+	addressId: string,
+	allowWildcard: boolean
+): Promise<GetAddressInformationModel> => {
+	const url = urlQueryBuilder(`${URL}/${addressId}/updates`, {
+		access_token: wdData.accessToken,
+	});
+
+	const res = await axiosConf.get(url);
+
+	return res.data;
+};
+
+/**
+ * List registered Addresses
+ *
+ * https://docs.wildduck.email/api/#operation/getAddresses
+ *
+ * @param query Partial match of an address
```

Comment:
No, these are query parameters, check the video I sent you and watch the section on **body parameters** and **query parameters**